task8/script.js

```diff
@@ -0,0 +1,58 @@
+const listItem = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const insertLoading = () => {
+  const div = document.getElementById("div");
+  const img = new Image();
+  img.src = "loading-circle.gif";
+  img.id = "loadingImg";
+  loading.appendChild(img);
```

Comment:
MUST
I am getting an error.
Please check it in the console.
Error Description: `index.js:11 Uncaught ReferenceError: loading is not defined`

Reply:
I forgot to change it when I changed the function name.
I would like to get into the habit of checking for errors by outputting to the console.

commit: 152fa02